### DISCUSSION TITLE
updateShopReceipt EndPoint Question

### DISCUSSION THREAD

**Discussion by LouGibson** [etsy/open-api#844] [Q&A] [answered]
Hello all,

I'm looking for a way to exchange messages between buyers and myself - the seller. If I update "message_from_seller" using updateShopReceipt is this relayed to the buyer via an email?  

Thanks in advance.
Lou

**Comment by Gareth064** [comment, marked as answer]
Hi 

No it won't email them.

There is no way to communicate with buyers using the API.

It has been requested in a previous issue to add to the API the ability to send and receive messages (like in the app) but it doesn't appear to be considered as a future feature.

**Reply by erichlin1** [reply]
Hi Gareth,

I have a follow-up question to LouGibson's initial question regarding 'message_from_seller'. When a buyer completes a transaction, Etsy emails the receipt to the buyer with the transaction details and a private message from the seller. From my understanding, this private message can be changed in the GUI 'info and appearance' tab or via the API, 'message_from_seller'. is that right? 

if so, is it possible to change the message -- to include order details pertaining to the buyer, review or survey links -- via the API, 'message_from_seller', before it's sent to the buyer?

Thanks in advance!

**Reply by Gareth064** [reply]
Hi @ericlinsd 

The message from the seller can't be manipulated before it's sent to a buyer on a per sale basis.

All buyers will receive the same message depending what it is set to at the time of a purchase.

To update this message via the API you need to use the updateShop endpoint. You will see 2 parameters related to this, sale_message and digital_sale_message.

**Comment by LouGibson** [comment]
Thanks Gareth, much appreciated.
